scripts/seed.bat

```diff
@@ -6,3 +6,4 @@ python manage.py loaddata profiles
 python manage.py loaddata emailaddresses
 python manage.py loaddata hackathons
 python manage.py loaddata showcase
+python3 manage.py loaddata showcase_site_settings
```

Comment:
any particular reason this one uses `python3` but the others above use `python`?

edit:
**scripts/seed.sh**
all of those use `python3` it seems... should **seed.bat** include `3` as well for each?

Reply:
Actually on Windows it needs to be `python`. I made that error before, so thanks for flaggin. Changed it there now.